packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3905,37 +3901,15 @@ export class ContainerRuntime
 	}
 
 	/**
-	 * * Forms a function that will request a Summarizer.
-	 * @param loaderRouter - the loader acting as an IFluidRouter
-	 * */
-	// eslint-disable-next-line import/no-deprecated
-	private formRequestSummarizerFn(loaderRouter: IFluidRouter) {
+	 * Forms a function that will create and retrieve a Summarizer.
+	 */
+	private formCreateSummarizerFn(loader: ILoader) {
 		return async () => {
-			const request: IRequest = {
```

Comment:
@NicholasCouri Wanted to see what your thoughts were on this change (to centralize creating summarizer in one place)

Reply:
That is actually great!!!

Reply:
I like the concept in general, but in this case we've marked ContainerRuntime's request function as deprecated anyway. Maybe we just mark the new exported const `"_summarizer"` as deprecated too, it can be removed "soon" (not really, I guess - gotta wait for LTS to hit internal.7.0)

Reply:
The const isn't exported from the package so no need to deprecate or be cautious about removal.

The purpose of my comment is not about public API surface and deprecation, but rather encapsulation and colocation of an already-internal concept.  This secret handshake is a common point of confusion, and we can help reduce that by encapsulating and colocating it as much as possible.

Currently `Summarizer.create()` only really exists as a test API - it's not called in production scenarios.  This change elevates it to a production-usage API.  However, if we're going to give it production-level scrutiny it doesn't really make sense to live on Summarizer as a public static API - its implementation only works because it's based on assumptions about how the ContainerRuntime internals work.

Probably a better approach would be to eliminate `Summarizer.create()` entirely, and move its implementation to `containerRuntime.ts` as a free function `TEST_requestSummarizer` in that file.  This would colocate the secret handshake in one file, reduce public API surface area on production classes, and probably even reduce bundle size.